insonmnia/worker/server.go

```diff
@@ -801,11 +801,6 @@ func (m *Worker) StopTask(ctx context.Context, request *pb.ID) (*pb.Empty, error
 	}
 
 	m.setStatus(&pb.TaskStatusReply{Status: pb.TaskStatusReply_FINISHED}, request.Id)
-	_, err := m.storage.Remove(containerInfo.TaskId)
```

Comment:
so, it will be stored forever?

Reply:
containers will be removed when the deal is [finished](https://github.com/sonm-io/core/blob/fix/keep-task-info-after-stop/insonmnia/worker/server.go#L410)